transitmatters/t-performance-dash
# fix(station-speeds): correct distances for glx => rest of green line

## Motivation


https://github.com/transitmatters/t-performance-dash/pull/939

The GLX distances were previously incorrect, so we had to disable showing speeds for that line.

## Changes

After we create the station graph, do a pass of the Floyd-Warshall algorithm to compute the shortest distance between station pairs. 

## Testing Instructions

Go to trip explorer for the Green Line. Ensure that the numbers for GLX => other green line stations look correct
